pdb2sql/StructureSimilarity.py

```diff
@@ -311,15 +310,20 @@ def compute_izone(self,cutoff=5.0,save_file=True,filename=None):
         Returns:
             dict: i-zone definition
         """
+        backbone = ['CA', 'C', 'N', 'O']
         sql_ref = interface(self.ref)
-        contact_ref = sql_ref.get_contact_atoms(cutoff=cutoff,extend_to_residue=True,return_only_backbone_atoms=True)
+        contact_ref = sql_ref.get_contact_atoms(cutoff=cutoff,extend_to_residue=True)
```

Comment:
why did you change the return only backbone ? In my experience it needs to be exactly like that to obtain the same results as PROFIT

Reply:
The updated code now has exactly the same results as PROFIT, which are tested in `test_StructureSimilarity.py`.

I removed the argument `return_only_backbone_atoms` from `get_contact_atoms()` is because 
1. this argument is quite confusing with the argument `only_backbone_atoms` that is to use only backbone atoms to calculate contact.
2. it is easier to get the backbone atoms by using `get()` after `get_contact_atoms()`